llumnix/arg_utils.py

```diff
@@ -16,48 +18,53 @@
 import argparse
 from typing import Tuple
 
-from llumnix.config import GlobalSchedulerConfig, MigrationConfig
-
+from llumnix.internal_config import GlobalSchedulerConfig, MigrationConfig
+from llumnix.config import LlumnixConfig, get_llumnix_config
+from llumnix.config.default import _C
 
 @dataclass
 class EngineManagerArgs:
-    launch_ray_cluster: bool = True
-    disable_init_instance_by_manager: bool = False
-    initial_instances: int = 1
-    disable_fixed_node_init_instance: bool = False
-
-    load_metric: str = 'remaining_steps'
-    polling_interval: float = 0.05
-
-    dispatch_policy: str = 'load'
-
-    enable_migration: bool = True
-    enable_defrag: bool = True
-    pair_migration_frequency: int = 1
-    pair_migration_policy: str = 'defrag_constrained'
-    migrate_out_threshold: float = 3.0
-    request_migration_policy: str = 'SJF'
-
-    enable_scaling: bool = False
-    min_instances: int = 1
-    max_instances: int = 1
-    scaling_interval: int = 10
-    scaling_policy: str = 'avg_load'
-    scale_up_threshold: float = 10
-    scale_down_threshold: float = 60
-
-    log_filename: str = "server.log"
-    disable_log_requests_manager: bool = False
-    log_instance_info: bool = True
-    profiling_result_file_path: str = ""
-
-    gpu_type: str = "a10"
-    migration_backend_init_timeout: float = 10.0
-    migration_backend: str = "rpc"
-    migration_cache_blocks: int = 512
-    migration_num_layers: int = 1
-    last_stage_max_blocks: int = 16
-    max_stages: int = 3
+    disable_init_instance_by_manager: bool = None
+    initial_instances: int = None
```

Comment:
Why not clear these `xxx=None`

Reply:
Displaying data types here is a good choice to avoid any ambiguity.